schema.py

```diff
@@ -21,22 +29,43 @@ def parse_args():
         type=str,
         help="This is the database schema, it is a json object.",
     )
+    parser.add_argument(
+        "-db",
+        "--database",
+        type=str,
+        help="The Database name",
+    )
+    parser.add_argument(
+        "-t",
+        "--table",
+        type=str,
+        help="The Table name",
+    )
+    parser.add_argument(
+        "-v",
+        "--value",
+        type=str,
+        help="The Value to add",
+    )
     return parser.parse_args()
 
 
 
 def create(schema):
     with open('database.json','r') as database:
         data = json.load(database)
-        if(os.path.exists(data[" database_name "])):
+        if(os.path.exists(data["database_name"])):
+            print("Already Exist")
             return
-        os.mkdir(data[" database_name "])
-        root_path = os.path.realpath(data[" database_name "])
-        for x in data[' Tables ']:
-            path = os.path.join(root_path, x[' name '])
+        os.mkdir(data["database_name"])
+        root_path = os.path.realpath(data["database_name"])
+        for x in data['Tables']:
+            path = os.path.join(root_path, x['name'])
             os.mkdir(path)
          
 
-args = parse_args()
-
-create(args.schema)
\ No newline at end of file
+cmd = input("Enter create : ")
+class CommandFactory:
```

Comment:
I recommend Factory has its own file and Every command has its own file

Reply:
Command Fatcory means it is a factory that creates an instance of ICommand classes. That's why its name is factory. 

Factory should have a method called `create` which returns ICommand